test/model/message_list_test.dart

```diff
@@ -985,6 +1022,45 @@ void main() {
         checkNotifiedOnce();
       }));
 
+      test('fetchOlder backoff start, _reset, fetchOlder backoff ends, move fetch finishes', () => awaitFakeAsync((async) async {
```

Comment:
nit: let's put this after both the existing "fetchOlder, _reset" test cases, instead of between the two of them. Those two are very similar and I feel like I want to read them together, comparing them.